docs/tutorial/agent-on-docker.md

```diff
@@ -0,0 +1,157 @@
+# Deploy Vald Agent on Docker
+
+Vald is designed and implemented based on Cloud-Native architecture.
+However, there may be cases that want to use only Vald Agent without Kubernetes.
+
+This article will show you how to deploy and run the Vald Agent on Docker.
+Fashion-mnist is used as an example dataset, same as [Get Stared](../tutorial/get-started.md)
```

Comment:
```suggestion
Fashion-mnist is used as an example dataset, same as [Get Stared](../tutorial/get-started.md) page.
```

Reply:
🤔

Reply:
I could not find another document such as "[title](internal link) page", but almost "[title](internal link)"
I don't think we have to fix, but should I fix it?
If there are any reasons to fix, please let me know about the reason. 🙏

Reply:
Thank you for your explanation, I agree with you. But could you fix it?
```suggestion
Fashion-mnist is used as an example dataset, same as [Get Started](../tutorial/get-started.md).
```